codespell_lib/data/dictionary.txt

```diff
@@ -9080,6 +9126,7 @@ oficially->officially
 ofo->of
 ofrom->from
 ofsetted->offsetted
+oft->of the
```

Comment:
Oft is a valid, although unlikely word. Often would be a likely and relevant replacement. When we can do spaces, I assume you'd find "oft he" here.

Reply:
A common mistake in LO sources is:
"elements oft he lists"

Anyway, I remove this.

Reply:
You should add that to the issue tracking multi-word replacements for the future though.